CHANGELOG.md

```diff
@@ -174,6 +174,139 @@
 
 <!-- NEW RELEASE NOTES ENTRY -->
 
+# v1.4.0 (draft)
+
+[Documentation](http://kubernetes.github.io) & [Examples](http://releases.k8s.io/release-1.4/examples)
+
+## Downloads
+
+binary | sha256 hash
+------ | -----------
+TODO [kubernetes.tar.gz](TODO) | `TODO`
+
+## Major Themes
+
+- **Simplified User Experience**
+  - Easier to get a cluster up and running (eg: `kubeadm`, intra-cluster bootstrapping)
+  - Easier to understand a cluster (eg: API audit logs, server-based API defaults)
+- **Stateful Appplication Support**
+  - Enhanced persistence capabilities (eg: `StorageClasses`, new volume plugins)
+  - New resources and scheduler features (eg: `ScheduledJob` resource, pod/node affinity/anti-affinity)
+- **Cluster Federation**
+  - Multi-Zone Ingress
+  - Expanded support for resources such as Namespaces, Events, ReplicaSets
+- **Security**
+  - Increased pod-level security granularity (eg: Container Image Policies, AppArmor and `sysctl` support)
+  - Increased cluster-level security granularity (eg: Access Review API)
+
+## Features
+
+This is the first release tracked via the use of the [kubernetes/features](https://github.com/kubernetes/features) issues repo.  Each Feature issue is owned by a Special Interest Group from [kubernetes/community](https://github.com/kubernetes/community)
+
+- **API Machinery**
+  - [alpha] Generate audit logs for every request user performs against secured API server endpoint. ([docs](http://kubernetes.io/docs/admin/audit/)) ([kubernetes/features#22](https://github.com/kubernetes/features/issues/22)) 
+  - [beta] `kube-apiserver` now publishes a swagger 2.0 spec in addition to a swagger 1.2 spec ([kubernetes/features#53](https://github.com/kubernetes/features/issues/53))
+- **Apps**
+  - [alpha] Introducing 'ScheduledJobs', which allow running time based Jobs, namely once at a specified time or repeatedly at specified point in time. ([docs](http://kubernetes.io/docs/user-guide/scheduled-jobs/)) ([kubernetes/features#19](https://github.com/kubernetes/features/issues/19)) 
+- **Auth**
+  - [alpha] Container Image Policy allows an access controller to determine whether a pod may be scheduled based on a policy ([docs](http://kubernetes.io/docs/admin/admission-controllers/#imagepolicywebhook)) ([kubernetes/features#59](https://github.com/kubernetes/features/issues/59))
+  - [alpha] Access Review APIs expose authorization engine to external inquiries for delegation, inspection, and debugging ([docs](http://kubernetes.io/docs/admin/authorization/)) ([kubernetes/features#37](https://github.com/kubernetes/features/issues/37))
```

Comment:
API Server endpoints to perform access control checks and subject access checks without direct knowledge of the backing authorization engine.

(is there a clearer description than "subject access checks"?) cc @erictune